Documentation/cmdref/cilium-agent.md

```diff
@@ -226,8 +226,10 @@ cilium-agent [flags]
       --hubble-dynamic-metrics-config-path string                 Filepath with dynamic configuration of hubble metrics.
       --hubble-event-buffer-capacity int                          Capacity of Hubble events buffer. The provided value must be one less than an integer power of two and no larger than 65535 (ie: 1, 3, ..., 2047, 4095, ..., 65535) (default 4095)
       --hubble-event-queue-size int                               Buffer size of the channel to receive monitor events.
+      --hubble-export-aggregation-interval duration               Interval at which to aggregate before exporting Hubble flows. 0s disables aggregation.
```

Comment:
optional nit: Please move the cmdref changes into the commit which actually adds the flags. That way the linters are happy on every individual commit.